external/fv3fit/fv3fit/emulation/layers/architecture.py

```diff
@@ -178,3 +258,88 @@ def NoWeightSharingSLP(
             tf.keras.layers.Flatten(),
         ]
     )
+
+
+class StandardOutputConnector(tf.keras.layers.Layer):
```

Comment:
Can you rename `Connector` to `Layer`...these seem like synonyms in this context. I'd also make these `Connector` classes private APIs. They don't seem optimally composable...a better possible API would be an object that could be used like this: 
```
DictLayer({a: Dense(10), b: Dense(10)}(x)
```
but I am not suggesting you actually do that.

However to support future refactors, you should re-target any coverage provided by the tests of `StandardOutputConnector` and `RNNOutputConnector` to focus on the top-level and `HybridRNN` and  `RNN` classes, respectively.